client/modules/IDE/components/EditorAccessibility.stories.jsx

```diff
@@ -0,0 +1,19 @@
+import EditorAccessibility from './EditorAccessibility';
+
+export default {
+  title: 'IDE/EditorAccessibility',
+  component: EditorAccessibility
+};
+
+export const Default = {
+  args: {
+    lintMessages: [
+      {
+        severity: 'warn',
+        line: '10',
+        message: 'Warning Foo',
+        id: '123'
+      }
+    ]
+  }
+};
```

Comment:
I'm not sure what makes sense for a story here, as this component is hidden/invisible.  We would want to review the text that we are outputting (I recall it being not quite right at the moment).  We might want to create an export which has the text contents of the component with no styling.

Reply:
I'm not terribly familiar with the components so I wasn't sure if it made sense having this here.

Probably makes sense to remove it form as a story and rely on unit tests to qualify behaviour if its not a visual thing.